rust/benchmarks/Cargo.toml

```diff
@@ -30,5 +30,5 @@ arrow = { path = "../arrow" }
 parquet = { path = "../parquet" }
 datafusion = { path = "../datafusion" }
 structopt = { version = "0.3", default-features = false }
-tokio = { version = "0.3", features = ["macros", "rt", "rt-multi-thread"] }
+tokio = { version = "0.2", features = ["macros"] }
```

Comment:
I think you need to revert to "rt-core", "rt-threaded" ?

Reply:
I don't think those are features in tokio 0.2 (annoyingly):

If you add them, you get the following error

```
diff --git a/rust/datafusion/Cargo.toml b/rust/datafusion/Cargo.toml
index ab11a9520..bc7b67808 100644
--- a/rust/datafusion/Cargo.toml
+++ b/rust/datafusion/Cargo.toml
@@ -57,7 +57,7 @@ chrono = "0.4"
 async-trait = "0.1.41"
 futures = "0.3"
 pin-project-lite= "^0.2.0"
-tokio = { version = "0.2", features = ["macros"] }
+tokio = { version = "0.2", features = ["macros", "rt", "rt-multi-thread"] }
 
 [dev-dependencies]
 rand = "0.7"
```


```
error: failed to select a version for `tokio`.
    ... required by package `datafusion v3.0.0-SNAPSHOT (/Users/alamb/Software/arrow2/rust/datafusion)`
versions that meet the requirements `=0.2.23` are: 0.2.23

the package `datafusion` depends on `tokio`, with features: `rt` but `tokio` does not have these features.
```

Reply:
Thanks for the tip @Dandandan  -- it is strange. It seems to compile for me locally without doing this -- I am re-building from scratch and will also see what happens on CI tests.

Reply:
yes you are right :) -- fixing